apps/api/docs/scripts/helper-unzip-gz-files.py

```
@@ -0,0 +1,18 @@
+import gzip
+import shutil
+import os
+
+def unzip_gz_files(pattern="../../logs/error*.log.gz"):
+    for file in os.listdir(pattern):
+        if file.endswith(".gz"):
+            gz_path = os.path.join(pattern, file)
+            output_path = os.path.join(pattern, file[:-3])  # Remove .gz extension
+
+            with gzip.open(gz_path, 'rb') as f_in:
+                with open(output_path, 'wb') as f_out:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)